### DISCUSSION TITLE
Make error

### DISCUSSION THREAD

**Discussion by softodnamra** [processone/ejabberd#3616] [Questions] [answered]
hello:
I have this error in Fedora Server 33

```
make
rm -rf deps/.got
rm -rf deps/.built
mkdir -p deps
/usr/lib64/erlang/bin/escript rebar get-deps && :> deps/.got
==> rel (get-deps)
==> ejabberd (get-deps)
Pulling base64url from {git,"https://github.com/dvv/base64url",{tag,"1.0.1"}}
ERROR: Rebar requires version {1,5} or higher of git to process {git,
                                                                 "https://github.com/dvv/base64url",
                                                                 {tag,
                                                                  "1.0.1"}}
ERROR: 'get-deps' failed while processing /opt/ejabberd: rebar_abort
make: *** [Makefile:128: deps/.got] Error 1
```

Me pueden ayudar por favor?

**Comment by badlop** [comment, marked as answer]
> ERROR: Rebar requires version {1,5} or higher of git

You have several solutions:
* Update Git to something newer than 1.5. For example, I have git version 2.20.1
* Instead of compiling from source code, download an installer or use the Feadora package https://docs.ejabberd.im/admin/installation/

**Reply by softodnamra** [reply]
Which one advises me to use the .run or the .rpm 
and Thanks

**Reply by badlop** [reply]
They contain exactly the same files

**Reply by softodnamra** [reply]
Hello and sorry
when I start the ejabberd /etc/init.d/ejabberd start it answers me this: 
Starting ejabberd...
This account is currently not available.
why?

**Reply by badlop** [reply]
How did you install ejabberd, exactly?

**Reply by softodnamra** [reply]
sudo dnf -y install /home/user/ejabberd-21.04-0.x86_64.rpm

**Reply by badlop** [reply]
Oh, I can't try that one because I use Debian. Check that script you want to use. Probably it expects you create a system account called "ejabberd" or similar, so ejabberd will run using that account instead of root.

**Reply by softodnamra** [reply]
OK Thanks for the help, I'll see what I do, and sorry for the delay is that here in Cuba the connection is difficult
